omnibus/config/software/datadog-agent.rb

```diff
@@ -89,7 +89,7 @@
     if ENV['DEBUG_CUSTOMACTION'] and not ENV['DEBUG_CUSTOMACTION'].empty?
       debug_customaction = "--debug"
     end
-    command "invoke customaction.build --major-version #{major_version_arg} #{debug_customaction} --arch=" + platform
```

Comment:
Unintended change? We should keep passing in `#{debug_customaction}` (or remove its definition altogether from the lines above if it's not needed).